src/orderbook/orderbooks.py

```diff
@@ -21,15 +23,17 @@ def peek(self):
 
     def size(self):
         return len(self)
+    
+    def is_empty(self):
```

Comment:
i prefer to use the self is / is not because it's more pythonic and recognized by pylint

Reply:
Ok if u want